compiler/qsc_frontend/src/lower/tests.rs

```diff
@@ -1436,7 +1436,7 @@ fn lambda_operation_closure() {
                         body: SpecDecl 9 [80-130]: Impl:
                             Block 10 [122-130] [Type Result]:
                                 Stmt 11 [124-128]: Expr: Expr 12 [124-128] [Type Result]: Call:
-                                    Expr 13 [124-126] [Type (Unit => Result)]: Var: Local 8
+                                    Expr 13 [124-126] [Type (Unit => Result is Param<0>)]: Var: Local 8
```

Comment:
Both of the changed tests in here reference an empty functor set. Do we have tests that reference non-empty functor set generic parameters?

Reply:
oh, actually, the one right after this just needs to be updated to include a usage of the passed in adjoint operation.

Reply:
It makes sense that the reference here is always a reference to the functor generic. And there are tests where that functor generic is non-empty, which is what I was asking about. Thanks!